src/constants.js

```diff
@@ -395,6 +396,7 @@ export const REPRESENTATIONS_RESPONSE_VIEWABLE: 'viewable' = 'viewable';
 export const SIDEBAR_VIEW_SKILLS: 'skills' = 'skills';
 export const SIDEBAR_VIEW_DETAILS: 'details' = 'details';
 export const SIDEBAR_VIEW_METADATA: 'metadata' = 'metadata';
+export const SIDEBAR_VIEW_METADATA_REDESIGN: 'metadata_redesigned' = 'metadata_redesigned';
```

Comment:
can we have this named `metadata_redesign` to match the variable name and a consistent naming convention